### ISSUE TITLE
My game using KivMob

### ISSUE THREAD

**Issue by RusikBuyadutov** [MichaelStott/KivMob#91]
https://play.google.com/store/apps/details?id=org.legerement.zaprotiv

Here is the Game 
Za Protiv or (Yes or No)

Big thanks to you! Ads not showing right now cuz of 30 days ban :( 

It is my first game btw

**Comment by akgupta0777** [comment]
Do ads are shown previously? Have you tested ads before and after launching to play store?

**Comment by RusikBuyadutov** [comment]
> 
> 
> Do ads are shown previously? Have you tested ads before and after launching to play store?

Yeah. I was playing with my friends before google launching.So my AdMob thought that I was scamming. Everything was perfect!

**Comment by akgupta0777** [comment]
Ohh That's why Google has given test ads 😁.

**Comment by Felipe738743872** [comment]
Haha, it happened to me too, I've been banned for 30 days :(

But it's weird because I don't have any users on my app and my AdMob account registered 0.45 USD ...
My account is new and I always use test ads
